Pear/Controller/HomeViewController.swift

```diff
@@ -94,6 +94,30 @@ class HomeViewController: UIViewController {
         navigationController?.setNavigationBarHidden(false, animated: false)
     }
 
+    private func showInAppFeedback() {
+        guard let netId = UserDefaults.standard.string(forKey: Constants.UserDefaults.userNetId) else { return }
+        NetworkManager.shared.getMatchHistory(netID: netId).observe { response in
+            switch response {
+            case .value(let value):
+                guard value.success else {
+                    print("Network error: could not get user match history")
+                    return
+                }
+                var previousMatchHistorySize = UserDefaults.standard.integer(forKey: Constants.UserDefaults.previousMatchHistorySize)
+                previousMatchHistorySize = previousMatchHistorySize == 0 ? 1 : previousMatchHistorySize
```

Comment:
im a little confused at why `previousMatchHistorySize` has to be 1 or bigger, Id add a docstring saying what the criteria is for it to give In App Feedback

Reply:
100% agree